new_platforms/samples/helloworld/HelloWorldHost/host.c

```diff
@@ -58,13 +62,87 @@ int main(int argc, const char* argv[])
     }
     if (hostResult != OE_OK)
     {
-        fprintf(stderr, "OCALL failed: result=%u", hostResult);
+        fprintf(stderr, "OCALL failed: result=%u\n", hostResult);
         goto exit;
     }
 
+    result = oe_get_report_v2(enclave, 0, NULL, 0, &report_buffer, &report_buffer_size);
```

Comment:
Rather than complicating what should be a simple helloworld sample, I would move this to a separate sample.
For comparison
openenclave/samples/local_attestation is separate from openenclave/samples/helloworld

Reply:
Will do.

Reply:
I have adopted the local_attestation sample from OE and tweaked it a bit to simplify and better showcase what we can do with CyReS. Let me know what you think.